workbench-core/example/app/src/routes/datasetRoutes.ts

```diff
@@ -60,21 +96,20 @@ export function setUpDSRoutes(
   router.post(
     '/datasets',
     wrapAsync(async (req: Request, res: Response) => {
-      processValidatorResult(validate(req.body, CreateDataSetSchema));
-
+      const validatedRequest = validateAndParse<CreateImportDataSet>(CreateImportDataSetParser, req.body);
       const authenticatedUser = validateAndParse<AuthenticatedUser>(AuthenticatedUserParser, res.locals.user);
 
+      const { roleToAssume, externalId, ...createDataSetParams } = validatedRequest;
+
+      const storageProvider =
```

Comment:
are you moving all dataset creation to the hosting account? Would be nice if we were testing both main account and hosting account.